### PR TITLE
fixed fatal error bug

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[Flutter] Fix logException "nonfatal" and segmentation problem](https://trello.com/c/THaw2LAB/1583-flutter-fix-logexception-nonfatal-and-segmentation-problem)

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Check out also the ios side

**Comment by peterBrxwn** [comment]
@ArtursKadikis done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ## xx.x.x
```
Comment (turtledreams) [inline CHANGELOG.md:1]: lets also add an entry about ios changes

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by turtledreams — APPROVED** [review]

**Comment by turtledreams** [comment]
we will have to add a test for this in a diffrent pr

**Review by ijunaid — APPROVED** [review]

**Pull request merged**